trade-finance-manager-ui/component-tests/case/activity/_macros/activity.component-test.js

```diff
@@ -1,4 +1,4 @@
-const pageRenderer = require('../../../component-tests/pageRenderer');
+const pageRenderer = require('../../../pageRenderer');
 
 const page = '../templates/case/activity/activity.njk';
 const render = pageRenderer(page);
```

Comment:
Has this file definitely been moved to the right location to mirror the structure of the templates? The template this file is testing is in `trade-finance-manager-ui\templates\case\activity\activity.njk` (i.e, not inside `\_macros`), but this test file is in `trade-finance-manager-ui\component-tests\case\activity\_macros\activity.component-test.js`.

Reply:
Moved the test under` trade-finance-manager-ui\component-tests\case\activity\activity.component-test.js`